.github/workflows/go.yaml

```diff
@@ -22,9 +22,10 @@ on:
 
 jobs:
   check:
-    uses: int128/go-workflows/.github/workflows/check.yaml@v0.3.0
+    uses: int128/go-workflows/.github/workflows/check.yaml@v0.4.0
     with:
-        go-version: 1.23.1
+        go-version-file: go.mod
```

Comment:
go.mod was still 1.22.5.
It has been updated by https://github.com/int128/kubelogin/pull/1148